src/BoundingBox2d.elm

```diff
@@ -922,3 +922,52 @@ is equivalent to
 translateIn : Direction2d -> Float -> BoundingBox2d -> BoundingBox2d
 translateIn direction distance boundingBox =
     translateBy (Vector2d.withLength distance direction) boundingBox
+
+
+{-| Expand the bounding box in all the directions by given distance;
+
+    expandBy_ : Float
+    expandBy_ =
+        3
+
+
+    --> BoundingBox2d.expandBy expandBy_ exampleBox
```

Comment:
I don't think the `expandBy_` variable is necessary, I think `BoundingBox2d.expandBy 3 exampleBox` would be shorter and arguably more readable...

Reply:
Done